.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+# General Information
+
+1. **What change/s does this pull request introduce? (Provide a detailed explanation)**
+
+2. **Does this pull request break something? What?**
+
+# Testing
+
+1. **Was this change tested locally?**
+
+2. **Was this change tested in staging?**
+
+3. **Was this change tested in production?**
```

Comment:
I think we could summarize these 3 into 1,

`How did you test the change? (environment, strategy, etc)`

Reply:
> I like them separate

I mean, there's no way that the change should be merged without being tested so I don't entirely see the benefit of having them separate.